tests/test_extensions_build.py

```diff
@@ -0,0 +1,38 @@
+import subprocess
+import sys
+import shutil
+
+from tests.conftest import TEST_ROOT, PROJECT_ROOT
+
+EXTENSIONS_BUILD_FILENAME = 'extensions_build_checks.py'
+
+
+def test_force_cuda_build(tmp_path):
+    venv_path = tmp_path / 'venv'
+    venv_path.mkdir()
+
+    ext_path = tmp_path / 'extensions'
+    export_env_variables = "export CUDA_VISIBLE_DEVICES='' export TORCH_EXTENSIONS_DIR={}".format(ext_path)
+
+    python_executable_with_venv = ". {0}/bin/activate && {1} && {0}/bin/python".format(venv_path, export_env_variables)
+
+    version_string = "{}.{}".format(sys.version_info[0], sys.version_info[1])
+
+    subprocess.call("virtualenv -ppython{} {}".format(version_string, venv_path), shell=True)
+
+    subprocess.run(
+        "{python} {nncf_repo_root}/setup.py develop".format(
+            python=python_executable_with_venv,
+            nncf_repo_root=PROJECT_ROOT),
+        check=True,
+        shell=True,
+        cwd=PROJECT_ROOT)
```

Comment:
This is mostly a copy-and-paste of a code from `test_install.py`. Reuse this setup code using pytest fixtures.

Reply:
It has some similar lines, but test_extensions_build.py uses other arguments. So I don't see a reason to try to unite test_install.py and test_extensions_build.py it will save a few lines but will demand to create new functions/file

Reply:
Creating a tmp_venv seems to be a common task in the install tests. Create a corresponding fixture and extract the tmp venv creation and activation code there.